scripts/deployment/deploy_bridge.js

```diff
@@ -16,18 +16,27 @@ async function main() {
     if (common.getBridgeAddress() && !process.env.ENFORCE_BRIDGE_DEPLOY)
         throw new Error('Bridge already deployed. aborting.');
 
+    const sendRewardsData = hardhat.ethers.utils.defaultAbiCoder.encode(
+            ["bytes32", "bytes32", "uint256"],
+            [hardhat.ethers.utils.formatBytes32String(config.bridge.sendRewards.toBlockchain),
+                hardhat.ethers.utils.formatBytes32String(config.bridge.sendRewards.toAccount),
+                hardhat.ethers.utils.parseEther(config.bridge.sendRewards.maxLockLimit)]
+    );
+
     log(`Deploying Bridge...`);
     const Bridge = await hardhat.ethers.getContractFactory('Bridge');
     const bridge = await Bridge.deploy(
-        config.bridge.maxLockLimit,
-        config.bridge.maxReleaseLimit,
-        config.bridge.minLimit,
-        config.bridge.limitIncPerBlock,
+        hardhat.ethers.utils.parseEther(config.bridge.maxLockLimit),
+        hardhat.ethers.utils.parseEther(config.bridge.maxReleaseLimit),
+        hardhat.ethers.utils.parseEther(config.bridge.minLimit),
+        hardhat.ethers.utils.parseEther(config.bridge.limitIncPerBlock),
         config.bridge.minRequiredReports,
-        config.bridge.commissionAmount,
+        hardhat.ethers.utils.parseEther(config.bridge.commissionAmount),
+        sendRewardsData,
         BBS_TOKEN_ADDRESS);
     common.etherscanLogContract(bridge.address, bridge.deployTransaction.chainId);
     fs.writeFileSync(common.bridgePath, bridge.address);
+    fs.writeFileSync(common.bridgeSendRewardsArgPath, sendRewardsData);
```

Comment:
This whole shifting between js config file and env files and stuff starts to smell a bit. I think you hinted at it when you asked for the review. Let's go over the mechanism on the phone.

Reply:
let's.